<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefUi</class>
 <widget class="QDialog" name="RefUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="findKeysLA">
         <property name="text">
          <string>&amp;Filter:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="refsL">
         <property name="text">
          <string>&amp;In[[buffer]]:</string>
         </property>
         <property name="buddy">
          <cstring>bufferCO</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QHBoxLayout" name="filterBarL"/>
         </item>
         <item>
          <widget class="QCheckBox" name="csFindCB">
           <property name="toolTip">
            <string>Filter case-sensitively</string>
           </property>
           <property name="text">
            <string>Case Sensiti&amp;ve</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="bufferCO">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The (sub-)document from which the available labels are displayed</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Available &amp;Labels:</string>
     </property>
     <property name="buddy">
      <cstring>refsTW</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTreeWidget" name="refsTW">
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="SortLA">
       <property name="text">
        <string>So&amp;rt:</string>
       </property>
       <property name="buddy">
        <cstring>sortingCO</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sortingCO">
       <property name="toolTip">
        <string>Sorting of the list of available labels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="groupCB">
       <property name="toolTip">
        <string>Group the list of available labels by prefix (e.g. &quot;sec:&quot;)</string>
       </property>
       <property name="text">
        <string>Grou&amp;p</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="referenceL">
       <property name="text">
        <string>Sele&amp;cted Label:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>referenceED</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="referenceED">
       <property name="toolTip">
        <string>Select a label from the list above or enter a label manually</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="gotoPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Jump to the selected label</string>
       </property>
       <property name="text">
        <string>&amp;Go to Label</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="typeLA">
       <property name="text">
        <string>Reference For&amp;mat:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>typeCO</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeCO">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Adjust the style of the cross-reference</string>
       </property>
       <item>
        <property name="text">
         <string>&lt;reference&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(&lt;reference&gt;)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;page&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>on page &lt;page&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;reference&gt; on page &lt;page&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Formatted reference</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Textual reference</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="nameL">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>nameED</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameED">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="updatePB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Update the label list</string>
       </property>
       <property name="text">
        <string>&amp;Update</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okPB">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyPB">
       <property name="text">
        <string>&amp;Apply</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closePB">
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>refsTW</tabstop>
  <tabstop>csFindCB</tabstop>
  <tabstop>groupCB</tabstop>
  <tabstop>referenceED</tabstop>
  <tabstop>typeCO</tabstop>
  <tabstop>nameED</tabstop>
  <tabstop>updatePB</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>applyPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
